roda-app.rb

```diff
@@ -5,16 +5,19 @@
 require 'set'
 require 'rack/csrf'
 
-# require_relative 'configurations/warden'
 Dir["./helpers/*.rb"].each {|file| require file }
 require_relative 'models'
-Dir["./services/*.rb"].each {|file| require file }
+Dir[File.join("./services", "**/*.rb")].each do |f|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.